webapp/components/_new/generic/TabNavigation/TabNavigation.vue

```diff
@@ -0,0 +1,82 @@
+<template>
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-09-23T06:46:39Z" title="Wednesday, September 23rd 2020, 8:46:39 am +02:00">Sep 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,65 @@
+<template>
```


Cool, that you implement a tab component! 😍

This component looks a bit technical. 😉

We use a kind of similar thing in the user profile to switch between `posts`, `commented`, and `shouted`. That would be a nice design. Or? @Mogge